Multisig/Logic/Models/AppSettings.swift

```diff
@@ -25,9 +26,20 @@ extension AppSettings {
     static var termsAccepted: Bool
 
     @AppSetting(\.trackingEnabled)
+    private static var _trackingEnabled: Bool
```

Comment:
maybe let's name it `trackingEnabledSetting` as the underscoring is not by the coding guidelines we adopted

Reply:
changed